pkg/registry/thirdpartyresourcedata/codec.go

```diff
@@ -220,6 +220,9 @@ func encodeToJSON(obj *experimental.ThirdPartyResourceData) ([]byte, error) {
 	return json.Marshal(objMap)
 }
 
+func (t *thirdPartyResourceDataCodec) Version() string {
+	return "ThirdpartyResourceDataCodec does not have a version"
```

Comment:
I suspect this is not true, @brendandburns may be able to tell you how to figure it out.